src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -53,15 +54,15 @@ static AuthorizationClient()
         }
 
         /// <summary>
-        /// Creates PoliciesClient using AzureContext instance.
+        /// Creates AuthorizationClient using AzureContext instance.
         /// </summary>
         /// <param name="context">The AzureContext instance</param>
         public AuthorizationClient(IAzureContext context)
         {
             ActiveDirectoryClient = new ActiveDirectoryClient(context);
             AuthorizationManagementClient = AzureSession.Instance.ClientFactory
 #if !NETSTANDARD
-                .CreateClient<AuthorizationManagementClient>
+                .CreateArmClient<AuthorizationManagementClient>
```

Comment:
After referenced the new package, if we use 'CreateClient' , there will be throw error when we build in VS.